examples/mcts_example.py

```diff
@@ -0,0 +1,396 @@
+    opp_kos = len(game.get_knocked_out(opp_team))
+    own_cas = len(game.get_casualties(own_team))
+    opp_cas = len(game.get_casualties(opp_team))
+    own_stunned = len([p for p in game.get_players_on_pitch(own_team, up=False) if p.state.stunned])
+    opp_stunned = len([p for p in game.get_players_on_pitch(opp_team, up=False) if p.state.stunned])
+    own_down = len([p for p in game.get_players_on_pitch(own_team, up=False) if not p.state.stunned])
+    opp_down = len([p for p in game.get_players_on_pitch(opp_team, up=False) if not p.state.stunned])
+    own_ejected = len(game.get_dungeon(own_team))
+    opp_ejected = len(game.get_dungeon(opp_team))
+    own_has_ball = False
+    opp_has_ball = False
+    ball_carrier = game.get_ball_carrier()
+    if ball_carrier is not None:
+        own_has_ball = 1 if ball_carrier.team == own_team else 0
+        opp_has_ball = 1 if ball_carrier.team == opp_team else 0
+    own = own_score/10 + own_has_ball/20 - (own_cas + own_ejected)/30 - own_kos/50 - own_stunned/100 - own_down/200
+    opp = opp_score/10 + opp_has_ball/20 - (opp_cas + opp_ejected)/30 - opp_kos/50 - opp_stunned/100 - opp_down/200
+    if game.state.game_over:
+        if game.get_winner() == agent:
+            own += 0.1
+        elif game.get_winner() is None:
+            opp += 0.1
+    return 0.5 + own - opp
+
+
+class Node:
+
+    def __init__(self):
+        self.evaluations = []
+
+    def num_visits(self):
+        return len(self.evaluations)
+
+    def visit(self, score):
+        self.evaluations.append(score)
+
+    def score(self):
+        return np.average(self.evaluations)
+
+    def print(self):
+        raise NotImplementedError
+
+    def opp(self):
+        raise NotImplementedError
+
+    def depth_stats(self):
+        pass
+
+
+class ActionNode(Node):
+
+    def __init__(self, game, hash_key, opp=False):
+        super().__init__()
+        self.hash_key = hash_key
+        self.available_actions = self._extract_actions(game)
+        self.children: List[ChangeNode] = []
+        self._opp = opp
+        self.terminal = game.state.game_over
+
+    def opp(self):
+        return self._opp
+
+    def _extract_actions(self, game):
+        actions = []
+        for action_choice in game.get_available_actions():
+            if action_choice.action_type == botbowl.ActionType.PLACE_PLAYER:
+                continue
+            if action_choice.action_type == botbowl.ActionType.END_SETUP:
+                continue
+            if len(action_choice.players) > 0:
+                for player in action_choice.players:
+                    actions.append(Action(action_choice.action_type, position=None, player=player))
+            elif len(action_choice.positions) > 0:
+                for position in action_choice.positions:
+                    actions.append(Action(action_choice.action_type, position=position))
+            else:
+                actions.append(Action(action_choice.action_type))
+        return actions
```

Comment:
Could consider shuffling the action list as simple a remedy to home playing better than away.  
```python 
random.shuffle(actions)
return actions
```

Reply:
I agree this is a decent approach but it would likely reduce the performance when playing as home. I like keeping these results because it highlights that a strong MCTS implementation needs proper action policy or ordering.